wire-ios/Wire-iOS/Sources/UserInterface/Settings/ChangingDetails/ConfirmEmailViewController.swift

```diff
@@ -52,13 +56,20 @@ extension UITableView {
     }
 }
 
+// MARK: - ConfirmEmailViewController
+
 final class ConfirmEmailViewController: SettingsBaseTableViewController {
+
+    // MARK: - Properties
+
     fileprivate weak var userProfile = ZMUserSession.shared()?.userProfile
     weak var delegate: ConfirmEmailDelegate?
-
+    typealias SettingsAccountSectionEmail = L10n.Localizable.Self.Settings.AccountSection.Email.Change
```

Comment:
shouldn't that type contain some keyword pointing it's about localization/strings?

Reply:
Good point! What would be your suggestion?

Reply:
I don't know if there is any convention in the repo, but maybe something like,: 
`SettingsAccountSectionEmailLocalizable` or if that's too long `SettingsEmailLocalizable` ?

Reply:
I don't think we have any convention at the moment. I think I will go with the first option!